rundeckapp/grails-app/conf/BuildConfig.groovy

```diff
@@ -123,7 +123,7 @@ grails.project.dependency.resolution = {
         compile("org.rundeck:rundeck-core:${rundeckVersion}")
         compile("org.rundeck:rundeck-storage-filesys:${rundeckVersion}")
 
-        runtime 'postgresql:postgresql:9.1-901.jdbc4'
+        runtime 'postgresql:postgresql:42.0.0'
```

Comment:
note: i think the dependency group id changed to `org.postgresql` so it should be:

    runtime 'org.postgresql:postgresql:42.0.0'